mock/cpp_mock_scenarios/src/traffic_simulation_demo.cpp

```diff
@@ -30,10 +29,7 @@ class TrafficSimulationDemoScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit TrafficSimulationDemoScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "traffic_simulation_demo",
-      ament_index_cpp::get_package_share_directory("kashiwanoha_map") + "/map", "lanelet2_map.osm",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.